podvm/Makefile.inc

```diff
@@ -31,9 +31,11 @@ RUST_TARGET := $(RUST_ARCH)-unknown-linux-$(LIBC)
 
 CDH_RESOURCE_PROVIDER ?= kbs
 SEALED_SECRET ?= yes
+AGENT_POLICY ?= yes
```

Comment:
Should we make the use of OPA configurable? We could have a flag (e.g. `DEFAULT_POLICY`) that lets a user set a permissive policy.

Reply:
On one hand I want to avoid too many config options, otoh, it might make sense to have policy configurable.
So something like 
AGENT_POLICY : this is to enable/disable policy inclusion
DEFAULT_POLICY : this is to specify the policy file to be used as the default.  

Sounds ok ?

Reply:
Having that many configuration options on the image makes manual testing impractical (at least for me), but at least it's enabled by default. We can opt for only supporting and testing images that are built by the CI w/ a static configuration.